parser/match_operations.go

```diff
@@ -28,13 +28,19 @@ type AmountSign int
 
 const (
 	// AnyAmountSign is a positive or negative amount.
-	AnyAmountSign = iota
+	AnyAmountSign = 0
 
 	// NegativeAmountSign is a negative amount.
-	NegativeAmountSign
+	NegativeAmountSign = 1
 
 	// PositiveAmountSign is a positive amount.
-	PositiveAmountSign
+	PositiveAmountSign = 2
+
+	// PositiveOrZeroAmountSign is a positive or zero amount.
+	PositiveAmountSignOrZero = 3
```

Comment:
Comment (`PositiveOrZeroAmountSign `) and constant names (`PositiveAmountSignOrZero `) don't match.

I would personally prefer `PositiveOrZeroAmountSign`, for naming consistency, even though zero is not a sign.